test/verify/check-machines-dbus

```diff
@@ -264,6 +264,28 @@ class TestMachinesDBus(machineslib.TestMachines):
         # Attach some interface
         m.execute("virsh attach-interface --persistent VmNotInstalled bridge virbr0")
 
+        # Change the os boot firmware configuration
+        if m.image in ['fedora-31', 'fedora-testing', 'debian-testing']:
+            b.wait_in_text("#vm-VmNotInstalled-firmware", "BIOS")
+            b.click("#vm-VmNotInstalled-firmware")
+            b.wait_present(".modal-body")
+            b.select_from_dropdown("select", "UEFI")
+            b.click("#firmware-dialog-apply")
+            b.wait_not_present(".modal-body")
+            b.wait_in_text("#vm-VmNotInstalled-firmware", "UEFI")
+
+            # Temporarily delete the OVMF binary and check the firmware options again
+            m.execute("mv /usr/share/edk2/ovmf/OVMF_CODE.fd /tmp && mv /usr/share/edk2/aarch64/QEMU_EFI-pflash.raw /tmp/")
```

Comment:
Is that really the same path on Fedora and Debian? I was quite sure it's a different one.

Reply:
Fixed.  You are right - the test got it as well